tutorials/nats-rdb-offload/index.mdx

```diff
@@ -0,0 +1,229 @@
+---
+meta:
+  title: Leverage on NATS Messaging Pub/Sub capabilities to offload your Managed Database.
+  description: Learn how to construct a pub/sub architecture that can help you manage the load coming to your database. Dive into this Python tutorial for step-by-step guidance on asynchronous data processing.
+content:
+  h1: Leverage on NATS Messaging Pub/Sub capabilities to offload your Managed Database.
+  paragraph: Learn how to construct a pub/sub architecture that can help you manage the load coming to your database. Dive into this Python tutorial for step-by-step guidance on asynchronous data processing.
+categories:
+  - functions
+  - messaging
+  - postgresql-and-mysql
+tags: terraform scraping-architecture
+dates:
+  validation: 
+  posted: 
+---
+
+## Introduction
+
+In this tutorial, we show how to set up a system that receives a message from a publisher and sends it to a managed database through a buffer subscriber. To do so, we  use Scaleway serverless products and deploy two applications:
+ - A publisher application, that sends a message to a NATS stream created with Scaleway Messaging and Queuing.
+ - A subscriber application, that consumes messages published to the stream and then writes the data into a Scaleway Managed Database.
+
+<Lightbox src="scaleway-scraping-architecture.webp" alt="" />
```

Comment:
```suggestion
```

Reply:
<img width="1502" alt="DB_offload_usecase" src="https://github.com/scaleway/docs-content/assets/133798561/4020313a-d22b-4c99-896d-fc67a0feeb1e">

A scheme of the architecture